quickstart_lists/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains 'starter lists' of packages for bootstrapping and syncing on-prem-builders.
+
+Background: Historically we bootstraped on-prem-builders by downloading all the packages in 'core'
+for all targets. That amounted to about 15GB, and was both too much and too little, in that many of
+the packages weren't needed, and for many patterns (effortless) other origins were needed.
+
```

Comment:
Maybe add a transition sentence or paragraph that calls out the new strategy instead of jumping right into naming pattern.